bedrock/mozorg/blocks/advertising.py

```diff
@@ -0,0 +1,147 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from wagtail import blocks
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail_link_block.blocks import LinkBlock
+
+CAPTION_TEXT_FEATURES = [
+    "bold",
+    "italic",
+    "link",
+    "superscript",
+    "subscript",
+    "strikethrough",
+    "link",
+]
+
+
+SOCIAL_MEDIA_ICON_CHOICES = [
+    ("linkedin", "LinkedIn"),
+    ("tiktok", "TikTok"),
+    ("spotify", "Spotify"),
+    ("twitter", "Twitter"),
```

Comment:
I thought we weren't using Twitter for comms anymore - eg it's no longer in the Mozilla.org footer. 

But if we are supporting linkss to it, it should be phrased as `X` not `Twitter`